src/components/Overlay/Overlay.scss

```diff
@@ -0,0 +1,15 @@
+@import '~om';
+
+.Overlay {
+  @include stack(2);
+
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: color('snow', 0.9);
+  justify-content: center;
+  align-items: center;
+  color: color('ink');
```

Comment:
I don't know if this is wise whatever but if you *want* to actually blur stuff under it you could use

```css
backdrop-filter: blur(1em); // or whatever value for blurriness
```

Reply:
It's not *super* well supported but for browsers that don't have it they'd just see the normal alpha overlay.